### PR TITLE
Add Managed Identity Test

### PR DESCRIPTION
**Pull request by mcgallan**
Add Test for [this folder](https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/azure/resource-manager/models/common-types/managed-identity) in Cadl Ranch

### CODE REVIEW

File: test/CadlRanchProjects/azure/resource-manager/models/common-types/ci.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
```
Comment (pshao25) [inline test/CadlRanchProjects/azure/resource-manager/models/common-types/ci.yml:2]: where does this file come from?

File: package-lock.json [hunk 2]
```diff
@@ -216,10 +226,11 @@
       }
     },
     "node_modules/@azure-tools/cadl-ranch-coverage-sdk/node_modules/@types/node": {
-      "version": "18.19.40",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-18.19.40.tgz",
-      "integrity": "sha512-MIxieZHrm4Ee8XArBIc+Or9HINt2StOmCbgRcXGSJl8q14svRvkZPe7LJq9HKtTI1SK3wU8b91TjntUm7T69Pg==",
+      "version": "18.19.41",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-18.19.41.tgz",
```
Comment (pshao25) [inline package-lock.json:230]: Revert changes in this file

Comment (mcgallan) [reply package-lock.json]: Fixed

File: test/CadlRanchProjects/azure/resource-manager/models/common-types/ci.yml (hunk 1 above)
Comment (mcgallan) [reply test/CadlRanchProjects/azure/resource-manager/models/common-types/ci.yml]: This file was generated before the first version of Azure's namespace was changed. It should have been deleted after the namespace was changed to _Azure, and it has now been removed.

**Review by pshao25 — APPROVED** [review]

**Pull request merged**